arch/arm/boot/dts/overlays/imx477-overlay.dts

```diff
@@ -5,6 +5,6 @@
 
 #include "imx477_378-overlay.dtsi"
```

Comment:
That was just a typo in the comment then - there is no `imx477_378.dtsi`.

I think it's the asymmetry that is most confusing; 477/378 have the following files:
```
imx378-overlay.dts
imx477-overlay.dts
imx477.dtsi
imx477_378-overlay.dtsi
```
while 290/327 have:
```
imx290-overlay.dts
imx290_327.dtsi
imx290_327-overlay.dtsi
```

Reply:
It looks like imx327-overlay.dts never got created in the past, partly as the driver didn't get updated to have the extra compatible string. I'll sort that out.
imx378 did need about 4 different registers set cf imx477, hence the driver did get updated.

I'll rename imx477.dtsi to imx477_378.dtsi so that we at least keep the same distinction over whether it is a file common to multiple devices, or just one.